CHANGELOG.md

```diff
@@ -20,6 +20,7 @@ See the [CONTRIBUTING guide](./CONTRIBUTING.md#Changelog) for instructions on ho
 ### Enhancements
 ### Bug Fixes
 ### Infrastructure
+- Github ci-runner Node.js issue fix
```

Comment:
change log should have PR link. Please refer https://github.com/opensearch-project/geospatial/blob/2.9/CHANGELOG.md

Reply:
Done.